### PR TITLE
Deploy portal with and without lorawan

### PR DESCRIPTION
**Pull request by hocinehacherouf**
## Description

What's new?

- Implement #826 

## What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Tests
- [ ] Other

To Test this feature, we could use this link : [https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FCGI-FR%2FIoT-Hub-Portal%2Fmain%2Ftemplates%2Fazuredeploy.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FCGI-FR%2FIoT-Hub-Portal%2Ffeature%2F826_deploy_without_lora%2Ftemplates%2FazuredeployUI.json](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FCGI-FR%2FIoT-Hub-Portal%2Fmain%2Ftemplates%2Fazuredeploy.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FCGI-FR%2FIoT-Hub-Portal%2Ffeature%2F826_deploy_without_lora%2Ftemplates%2FazuredeployUI.json)

### CODE REVIEW

File: templates/azuredeploy.json [hunk 1]
```diff
@@ -86,14 +89,23 @@
             "metadata": {
                 "description": "SPI Dev version for x86 based gateway"
             }
+        },
```
[the remark this answers is not in the document]
Comment (hocinehacherouf) [reply templates/azuredeploy.json:88]: Done

File: templates/azuredeployUI.json [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+                        }
+                    ]
+                },
+				{
+					"name": "commonStep",
+					"label": "Common Settings",
+					"elements": [
+						{
+							"name": "uniqueSolutionPrefix",
+							"type": "Microsoft.Common.TextBox",
```
Comment (kbeaugrand) [inline templates/azuredeployUI.json:29]: Should validate the input : 
- only alpha numeric characters 
- max 16 characters

**Review by kbeaugrand — CHANGES_REQUESTED** [review]

**Comment by kbeaugrand** [comment]
Small issue when selecting LoRaWAN feature +not deploy edge device : 

image

**Comment by kbeaugrand** [comment]
> Small issue when selecting LoRaWAN feature +not deploy edge device :
> 
> image

Same when not-selecting lorawan : 

image

**Review by kbeaugrand — APPROVED** [review]

**Pull request merged**